cmd/awscollector/Dockerfile

```
@@ -4,42 +4,22 @@
 ################################
 FROM alpine:latest as build 
 
-ENV GOPROXY=direct
-
 # update cert
 RUN apk --update add ca-certificates
 
-# install libs
-RUN apk add --update bash git make build-base
-
-# config go
-COPY --from=golang:1.17-alpine /usr/local/go/ /usr/local/go/
-ENV PATH="/usr/local/go/bin:${PATH}"
-
 WORKDIR /workspace
 
 # copy artifacts
 COPY cmd  /workspace/cmd
```

Is this just to copy the script? If so, make it more explicit.
```suggestion
COPY cmd/awscollector/copy_docker_binary.sh  /workspace/
```